plugins/lando-services/services/mariadb/builder.js

```diff
@@ -8,12 +8,13 @@ module.exports = {
   name: 'mariadb',
   config: {
     version: '10.3',
```

Comment:
Should we bump the default too? 

10.5 was released on Dec 3, 2019. 

https://mariadb.com/kb/en/mariadb-1050-release-notes/